### ISSUE TITLE
nested loops not working

### ISSUE THREAD

**Issue by MiniXC** [MEH-Design/frix#61]
I could not pin this down exactly yet, but it seems that nested loops are only working when the first loop element of the outer loop has only one element.
**example**
# &#x2705;
Both lines are generated.
````json
"lines": [
    {
      "type": "contentline",
      "data": {
        "color": "green",
        "modules": [
          {
            "type": "titledbox",
            "data": {
              "heading": "Einzelberatung",
              "text": "Coaching und Lebensberatung in einem vertraulichen Rahmen.",
              "image": "https://cdn4.iconfinder.com/data/icons/file-extensions-1/64/pdfs-512.png"
            }
          }
        ]
      }
    },
    {
      "type": "contentline",
      "data": {
        "color": "beige",
        "modules": [
          {
            "type": "calltoaction",
            "data": {
              "borderimage": "resources/portrait.jpg",
              "heading": "Herzlich Willkommen!",
              "text": "Meine Angebote sind sehr vielfältig und doch haben sie alle eines gemeinsam - <br><strong>Achtsam mit sich selbst und anderen.</strong>",
              "button": {
                "text": "Mehr über mich erfahren..."
              }
            }
          },
          {
            "type": "review",
            "data": {}
          }
        ]
      }
    }
  ]
````
# &#x274E;
Only the first line is generated.
````json
  "lines": [
    {
      "type": "contentline",
      "data": {
        "color": "beige",
        "modules": [
          {
            "type": "calltoaction",
            "data": {
              "borderimage": "resources/portrait.jpg",
              "heading": "Herzlich Willkommen!",
              "text": "Meine Angebote sind sehr vielfältig und doch haben sie alle eines gemeinsam - <br><strong>Achtsam mit sich selbst und anderen.</strong>",
              "button": {
                "text": "Mehr über mich erfahren..."
              }
            }
          },
          {
            "type": "review",
            "data": {}
          }
        ]
      }
    },
    {
      "type": "contentline",
      "data": {
        "color": "green",
        "modules": [
          {
            "type": "titledbox",
            "data": {
              "heading": "Einzelberatung",
              "text": "Coaching und Lebensberatung in einem vertraulichen Rahmen.",
              "image": "https://cdn4.iconfinder.com/data/icons/file-extensions-1/64/pdfs-512.png"
            }
          }
        ]
      }
    }
  ]
````

**Comment by MiniXC** [comment]
The upper loop works with *more than one element*... So maybe this doesn't have to do with loops at all?
Maybe the empty ``data: { }`` attribute of the *review* element is the culprit?

**Comment by MiniXC** [comment]
*Update*: I just did a bit of troubleshooting and discovered that even with a "proper" data attribute, the issue remains.

**Comment by MiniXC** [comment]
This is a problem with the specific template, not frix. The ``iframe`` seems to consume everything that is placed after it ¯\_(ツ)_/¯

**Comment by MiniXC** [comment]
The iframe problem is addressed in #62
